tests/e2e-openshift/red-metrics/03-install-hotrod.yaml

```diff
@@ -0,0 +1,33 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: hotrod
+  labels:
+    app: hotrod
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: hotrod
+  template:
+    metadata:
+      labels:
+        app: hotrod
+    spec:
+      containers:
+        - name: hotrod
+          image: jaegertracing/example-hotrod:1.46.0
+          ports:
```

Comment:
I'm seeing `2023/07/06 12:59:53 Post "http://localhost:14268/api/traces": dial tcp [::1]:14268: connect: connection refused` in the logs of the hotrod pod.

Afaics
```
env:
  - name: OTEL_EXPORTER_OTLP_ENDPOINT
    value: http://otel-collector:4318
```
is missing in the deployment.

Reply:
Ah sorry, I missed a part. I successfully tested this configuration a while ago:
```
      - image: jaegertracing/example-hotrod:1.46
        name: hotrod
        args:
          - all
          - --otel-exporter=otlp
        env:
          - name: OTEL_EXPORTER_OTLP_ENDPOINT
            value: http://otel-collector:4318
```